src/com/content-event/event-graded-game.js

```diff
@@ -0,0 +1,15 @@
+import {h, Component}					from 'preact/preact';
+import UIButtonLink from 'com/ui/button/button-link';
+
+export default class GradedGame extends Component {
+	render( props ) {
+		const {node} = props;
+		console.log(node);
```

Comment:
remove log

Reply:
Fixed (Going to write that to take note of what remains for me so I don't forget anything. Hope it's not too annoying).